### PR TITLE
[1LP][RFR] clone dialog is broken. uncollecting tests until those are fixed

### PR DESCRIPTION
**Pull request by izapolsk**
{{ pytest: --long-running cfme/tests/control/test_actions.py::test_action_cancel_clone cfme/tests/infrastructure/test_vm_clone.py::test_vm_clone}}

### CODE REVIEW

**Comment by lina-nikiforova** [comment]
@izapolsk should these blockers work only for `5.10`?
Also, this might be related https://bugzilla.redhat.com/show_bug.cgi?id=1622952

**Comment by izapolsk** [comment]
@lina-nikiforova, as I see in PRT. this issue is relevant to 5.10 only.
as for that bug, Harpreet left reference to it in the bug from PR. so, I'm aware of it. thanks.

File: cfme/tests/control/test_actions.py [hunk 1]
```diff
@@ -426,7 +426,7 @@ def test_action_prevent_event(request, vm, vm_off, policy_for_testing):
         pytest.fail("CFME did not prevent starting of the VM {}".format(vm.name))
 
 
-@pytest.mark.meta(blockers=[1439331])
+@pytest.mark.meta(blockers=[BZ(1439331)])
```
Comment (lina-nikiforova) [inline cfme/tests/control/test_actions.py]: Why do you keep his BZ? It's `CLOSED CURRENTRELEASE`

Comment (izapolsk) [reply cfme/tests/control/test_actions.py]: I just fixed bug. I didn't even check this. I think this is test owner responsibility.

**Comment by lina-nikiforova** [comment]
@izapolsk `test_action_cancel_clone` is failing for `5.9`. Is there the same bug and it should be blocked or is it another issue?

**Comment by izapolsk** [comment]
@lina-nikiforova, this 5.9 failure isn't related to that bug and caused by something else.

**Comment by digitronik** [comment]
As this PR under `blocker-only` label LGTM :+1:

**Pull request merged**